src/system/Form/RadioBoxGroup.js

```diff
@@ -94,16 +95,65 @@ const RadioOption = ( {
 	);
 };
 
-RadioOption.propTypes = {
+const ChipOption = ( {
+	defaultValue,
+	option: { value, label },
+	name,
+	disabled,
+	onChangeHandler,
+} ) => {
+	const checked = `${ defaultValue }` === `${ value }`;
+
+	return (
+		<Button
+			variant="text"
+			role="radio"
+			disabled={ disabled }
+			name={ name }
```

Comment:
What is this name property in Button? It looks like you are using label already.

Reply:
To get the name as `value` in the click handler (ie ` e.target.value`).

Reply:
Is there another way to do it?

Reply:
In the case of using the regular structure of the `Radio` component, the onChange, passes the event, and the `option` that you selected.

Here's an example:

```
const onChangeHandler = ( e: React.ChangeEvent< HTMLInputElement > ) => {
			const optionTriggered = options.find( option => {
				const optionValue = `${ option.value }`;
				const selectedOptionValue = `${ e.target.value }`;

				return optionValue === selectedOptionValue;
			} );

			if ( onChange ) {
				onChange( e, optionTriggered );
			}
		};
```

That way, you don't need value or name.